### PR TITLE
Close subscriber PC after a wait to aid in migration.

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

File: pkg/rtc/participant.go [hunk 1]
```diff
@@ -697,6 +699,54 @@ func (p *ParticipantImpl) Negotiate(force bool) {
...
+	p.lock.Lock()
+	if p.migrationTimer != nil {
+		p.migrationTimer.Stop()
+		p.migrationTimer = nil
+	}
+	p.lock.Unlock()
+}
+
+func (p *ParticipantImpl) MaybeStartMigration(force bool, onStart func()) bool {
+	if !force && p.TransportManager.HaveAllTransportEverConnected() {
```
Comment (boks1971) [inline pkg/rtc/participant.go:712]: Making sure that all transport is connected before allowing migration. We have seen cases where migration hobbles when all transport is not fully connected.

**Review by davidzhao — APPROVED** [review]
lg!

File: pkg/rtc/participant.go (hunk 1 above)
Comment (cnderrauber) [reply pkg/rtc/participant.go:712]: @boks1971  confused by the code and comment, if we don't want migrate unconnected participant, should this check be `if !force && !p.TransportManager.HaveAllTransportEverConnected` ?

Comment (boks1971) [reply pkg/rtc/participant.go:712]: Good catch @cnderrauber . Will make a PR to fix.

**Pull request merged**